### ISSUE TITLE
banshee.fm offline

### ISSUE THREAD

**Issue by Carbenium** [arfbtwn/banshee#67]
It seems the Banshee homepage banshee.fm isn't online anymore. The DNS records aren't set anymore.

As a consequence the online media features don't work anymore (Amazon, Miro Guide).

**Comment by njlittle** [comment]
Yeah, I noticed this morning trying to test your patch for the web view thinking my inability to zoom might have just been in the context panel.

I never build those extensions myself so I'm happy to drop support and remove them.

**Comment by arfbtwn** [comment]
Yeah, I noticed this morning trying to test your patch for the web view.

I never build those extensions myself so I'm happy to drop support and remove them.
